addons/binding/org.openhab.binding.exec/README.md

```diff
@@ -10,147 +10,93 @@ Currently, the binding supports a single type of Thing, being the `command` Thin
 
 The binding does not require any specific configuration.
 
+Note that the commands are executed in the context and with the privileges of the process running the java virtual machine. It is not advised to run the virtual machine as superuser/root. The execution of the commands is triggered by either sending a Command or State Update to the input Channel of the `command` Thing, if configured so, or by sending the ON Command to the run Channel of the `command` Thing (see below)
```

Comment:
We put this in the -**Linux:** section and added some more info. 
New is "The execution ..." which i think is not apropriately placed here, but a good additional information.
I would rephrase it and place it to .. maybe the channels? Under the tabular.
" The execution of the command can be triggerd by either sending the ON command to the run channel or by sending a command or state update to the input channel, when the thing is configured with runOnInput=true."

Reply:
I will stop here as this will just tripple the work. my suggestion to @kgoderis:
Try to first merge the most obvious things and then i am happy to review it again.